lib/Installation/Partitioner/LibstorageNG/v4/ExpertPartitionerController.pm

```diff
@@ -133,6 +137,37 @@ sub clone_partition_table {
     $self->get_expert_partitioner_page()->press_ok_clone();
 }
 
+sub create_msdos_partition_table {
+    my ($self) = @_;
+    $self->get_expert_partitioner_page()->go_top_in_system_view_table();
+    $self->get_expert_partitioner_page()->modify_create_new_partition_table();
+    $self->get_expert_partitioner_page()->select_msdos_partition_table();
+}
+
+sub add_primary_partition_msdos {
```

Comment:
I'm afraid we are facing some dumb issue here: AFAIS, you could achieve the same thing with add_partition_on_gpt_disk from lib/Installation/Partitioner/LibstorageNG/v3/ExpertPartitionerController.pm. the only problem is the naming :(. At least, you should use _add_partition from lib/Installation/Partitioner/AbstractExpertPartitionerController.pm which makes set_new_partition_size add_new_partition redondant here.